README.md

```diff
@@ -1 +1,225 @@
-# Exonum RFCs
\ No newline at end of file
+# Exonum RFCs
+[Exonum RFCs]: #exonum-rfcs
+
+Exonum RFC process is partially derived from [the Rust RFC process](https://github.com/rust-lang/rfcs/blob/master/README.md)
+
+Many changes, including bug fixes and documentation improvements can be
+implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put through a
+bit of a design process and produce a consensus among the Exonum community.
+
+The "RFC" (request for comments) process is intended to provide a consistent
+and controlled path for new features to enter the Exonum, so that all
+stakeholders can be confident about the direction the Exonum project is evolving
+in.
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+  - [Opening](#exonum-rfcs)
+  - [Table of Contents]
+  - [When you need to follow this process]
+  - [Before creating an RFC]
+  - [What the process is]
+  - [The RFC life-cycle]
+  - [Reviewing RFCs]
+  - [Implementing an RFC]
+  - [RFC Postponement]
+  - [Help this is all too informal!]
+
+## When you need to follow this process
+[When you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make "substantial" changes to
+Exonum core or the RFC process itself. What constitutes a
+"substantial" change is evolving based on community norms and varies depending
+on what part of the ecosystem you are proposing to change, but may include the
+following.
+
+- API changes.
```

Comment:
Inconsistent indentation with other lists. Also: what does "API changes" entail? Changes in the public interface of the `exonum` crate?